miso-web/src/main/webapp/scripts/warning_partition.js

```diff
@@ -36,21 +36,21 @@ WarningTarget.partition = {
           level: "important"
         }, {
           include: pool.duplicateIndices,
-          tableMessage: "(DUPLICATE INDICES)"
+          tableMessage: Constants.warningMessages.duplicateIndices
         }, {
           include: pool.nearDuplicateIndices && !pool.duplicateIndices,
-          tableMessage: "(NEAR-DUPLICATE INDICES)"
+          tableMessage: Constants.warningMessages.nearDuplicateIndices
```

Comment:
Parentheses have been removed from some warning messages, but not others. I'd be in favour of removing them from all - I think it's enough that they're coloured differently, and are usually in their own column too. Either way, they should be consistent

Reply:
Updated all to remove parentheses.